animation_nodes/sockets/object.py

```diff
@@ -59,6 +59,17 @@ def createObject(self):
             data = bpy.data.curves.new("Curve", "CURVE")
             data.dimensions = "3D"
             data.fill_mode = "FULL"
+        if type == "GPENCIL":
+            data = bpy.data.grease_pencils.new("ANGPencil")
+            scene = bpy.context.scene
+            scene.grease_pencil = data
```

Comment:
Why do we set annotations to this data block?

Reply:
Sorry, we do not need set annotations.